app/components/sdg/goals/target_component.rb

```diff
@@ -0,0 +1,22 @@
+class SDG::Goals::TargetComponent < ApplicationComponent
+  attr_reader :targets
+
+  def initialize(targets)
+    @targets = targets
+  end
+
+  def render?
+    feature?("sdg")
+  end
+
+  private
+
+    def type
+      return "global" if targets.model.name == "SDG::Target"
+      "local"
```

Comment:
```suggestion
      if targets.model.name == "SDG::Target"
        "global"
      else
        "local"
      end
```

It looks the same, but IMHO using `if` has a subliminal message meaning both routes are similar, while using `return` means something special/different happens when that condition is true.